test/cmdlineTests.sh

```diff
@@ -121,6 +122,7 @@ function test_solc_behaviour()
     local filename="${1}"
     local solc_args="${2}"
     local solc_stdin="${3}"
+    [ -z "$solc_stdin"  ] && solc_stdin="/dev/stdin"
```

Comment:
If `solc_stdin` is empty, doesn't it mean it does not matter. What is the purpose of adding `</dev/stdin` to the solc command?

Reply:
Because we redirect stdin in some cases and the `<` cannot be "read" from a variable content.

Reply:
Previously, `test_solc_behavior` did the following

```
if [[ -n "$solc_stdin"  ]]; then solc_command+=" <$solc_stdin"  ; fi
```

so it didn't change `solc_command` if `solc_stdin` were not set. So I'm wondering if it worked without the additional input redirect before, why the change now.

Reply:
The main change here is avoiding the use of `eval`. With `eval`, you can of course use `<`, but I'm a little uneasy as far as whitespace is concerned in connection with `eval`.

Reply:
Great! We had some discussions before putting it in already and I'm glad that it's gone again.